xbmc/guilib/imagefactory.cpp

```diff
@@ -37,9 +36,9 @@ IImage* ImageFactory::CreateLoader(const CURL& url)
 
 IImage* ImageFactory::CreateLoaderFromMimeType(const std::string& strMimeType)
 {
-  BinaryAddonBaseList addonInfos;
+  std::vector<AddonInfoPtr> addonInfos;
 
```

Comment:
Remove this empty line? At other code sequences similar to this one there is no blank line.